test_notebooks.py

```diff
@@ -15,7 +15,9 @@
 import nbformat
 import pint
 import pytest
-from git.cmd import Git
+
+from utils import find_files
```

Comment:
perhaps a relative import, not to confuse with https://pypi.org/project/utils?

Reply:
`attempted relative import with no known parent package`

Reply:
so maybe a less general name, like `devops_tests_commons`?

Reply:
maybe make a package from devops_tests?

Reply:
the way we use it, through `pytest` which scans directories for `test*.py` files, would ignore anything pip-installed, so making it an installable package would likely not have any added value... but just adding an `__init__.py` in the root dir (without `setup.py` or `pyproject.toml`) sounds reasonable.

Reply:
updated